### PR TITLE
Add Contact form section

### PR DESCRIPTION
**Pull request by Otromascomotu**
1. Use Flexbox to lay out the elements on the page.
2. Add a title of the section, "Contac Me", in html file with an h2 tag.
3. Add the contact form section with a basic template structure common.
4. Add some specific characteristics as:
- Name field accepts up to 30 chars.
- Email field accepts only emails.
- Message field accepts up to 500 chars.
- All fields have placeholder text.
- All fields have the correct type.
5. Add an image to the background of the contact form section.
6. Add the way to collect data in line 276 in html file using the Formspree service.
7. Add hover effect to the submit using an id.
8. Fix errors with Stylelint.
9. Add the value post in the method attribute in the tag form of the .css file.
10. Add styles to button interactions.

### CODE REVIEW

**Comment by Otromascomotu** [comment]
My project has passed all linters tests except Webhint's, but that's a GitHub platform fault, not mine.
Please read the announcement made on 06/13/2023 on #announcements on Microverse Slack regarding this.
Thanks for understanding.

File: index.html [hunk 1]
```diff
@@ -270,7 +270,28 @@ <h5>LET'S CONNECT</h5>
           <li class="skills-text">Skills</li>
             <img class="disabled-menu2" src="./imgs/disabled-menu/disabled-menu.svg" alt="disabled menu logo" />
       </ul>
+      </section>
+      <!-- Contact Form Section -->
+      <section id="section-contact-form" class="contact-form">
+        <form id="form" class="cform" action="https://formspree.io/f/mrgvnevk">
```
Comment (victorokoroji) [inline index.html]: - [x] Kindly ensure that your form has the method attribute with the value 'POST" as this is required as you will be sending data to your formspee. Without this, your form will not be submitted.
e.g
`<form id="form" class="cform" method="POST" action="https://formspree.io/f/mrgvnevk">`

image

File: index.html [hunk 2]
```diff
@@ -270,7 +270,28 @@ <h5>LET'S CONNECT</h5>
...
+            <input type="text" id="name" name="name" placeholder="Lorem Ipsum" maxlength="30" required />
+          </div>
+          <div class="form-group">
+            <input type="email" id="email" name="email" placeholder="loremipsum@gmail.com" required />
+          </div>
 
+          <div class="form-group">
+            <textarea id="message" name="message" placeholder="Write your message here..." maxlenght="500" required /></textarea>
+          </div>
+          <input id="submit" type="submit" value="Get in touch">
```
Comment (victorokoroji) [inline index.html:292]: - [x] For good user experience, Please ensure that you implement button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are styled as shown in the Figma template.

image

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @Otromascomotu, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Form validations are correctly applied ✅ 
- Form uses correct types and placeholder ✅ 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Otromascomotu) [reply index.html:292]: It's done! ✍️✅

File: index.html (hunk 1 above)
Comment (Otromascomotu) [reply index.html]: It's done! ✍️✅

**Comment by Otromascomotu** [comment]
My project has passed all linters tests except Webhint's, but that's a GitHub platform fault, not mine.
Please read the announcement made on 06/13/2023 on #announcements on Microverse Slack regarding this.
Thanks for understanding.

**Review by stanleySimeon — APPROVED** [review]
# Hi @Otromascomotu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good PR description
- Meaningful commit message

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**